odoolint/tests/test_convert_file.py

```diff
@@ -27,7 +29,7 @@ def filter(self, record):
         self.buffer.append(record.__dict__)
 
 
-class TestConvertFile(common.TransactionCase):
+class TestConvertFile(common.TransactionCase, unittest.TestCase):
```

Comment:
Did you mean this:
```bash
"""                                                                             
  3 The module :mod:`openerp.tests.common` provides unittest2 test cases and a few  
  4 helpers and classes to write tests.                                             
  5                                                                                 
  6 """  
import unittest2
```

I'm not pretty sure if we can use the `@unitttest` decorators but I will try it.